jobserver/views/errors.py

```diff
@@ -0,0 +1,66 @@
+from django.template.response import TemplateResponse
+
+
+def bad_request(request, exception=None):
```

Comment:
The exception parameter isn't being used, so can be removed. We can always add it back in if needed later. Same with the other functions below.

Reply:
The error handling views all have to accept a request and exception 🙂  (ie [handler400](https://docs.djangoproject.com/en/4.1/ref/urls/#django.conf.urls.handler400))